### ISSUE TITLE
Make openshift-ovirt role able to create an all-in-one inventory

### ISSUE THREAD

**Issue by rgolangh** [oVirt/ovirt-openshift-extensions#76]
**Is your feature request related to a problem? Please describe.**
The openshift-ovirt role under github.com/openshift/openshift-ansible  generates an inventory from created vms, but incase the inventory is made of one node, we don't assume its all in one. We need to add the right dirctives to it.

**Describe the solution you'd like**
When specifying an installation that is made of 1 VM, 1 node, which is both master and compute node, verify the inventory generated is like so:
```ini
[masters]
ocp-master

[etcd]
ocp-master

[nodes]
ocp-master openshift_schedulable=true ansible_host="{{ lookup('env', 'RHCI_ocp_master_IP') }}" openshift_node_group_name="node-config-all-in-one"
```

**Additional context**
Should the openshift-ansible add that lables automatically if there is a single node anyway?

**Comment by rgolangh** [comment]
@sradco please review

**Comment by rgolangh** [comment]
merged to openshift-ansible 3.11 release
